sdk/storage/Azure.Storage.Webjobs.Extensions.Queues/src/StorageQueuesWebJobsBuilderExtensions.cs

```diff
@@ -34,6 +35,7 @@ public static IWebJobsBuilder AddAzureStorageQueues(this IWebJobsBuilder builder
             // add webjobs to user agent for all storage calls
             DiagnosticsOptions.DefaultApplicationId = "AzureWebJobs";
 
+            builder.Services.AddAzureClients(_ => { });
```

Comment:
is this empty lambda necessary?

Reply:
Yeah, there is no AddAzureClients overload without a parameter but we need some services it adds.